boilerplate/app/screens/welcome-screen/welcome-screen.tsx.ejs

```diff
@@ -69,7 +69,12 @@ const CONTINUE_TEXT: TextStyle = {
   fontSize: 13,
   letterSpacing: 2,
 }
+
+<% if (props.useExpo && props.includeDetox) { -%>
+const FOOTER: ViewStyle = { backgroundColor: "#20162D", marginBottom: 64 }
```

Comment:
Not a huge deal but what is the purpose of adding the 64 margin at the bottom? If it is needed for detox can we add it even without to reduce the logic branches in this file?

Reply:
@rmevans9 It's in the PR description, but I think I agree that we can lose the `if` here and just always add it.